CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Owners (in alphabetical order)
+# Note: This is only for the notaryproject/notary repo and includes the Org level maintainers
+* @hukeping @jonnystoten @niazfk @priteshbandi @shizhMSFT @toddysm @vaninrao10 @yizha1
```

Comment:
For inactive repos without activities neither active repo maintainers in recent 1~2 years, should we prioritize the [archival process](https://github.com/notaryproject/.github/blob/main/REPOSITORIES.md#archiviation) instead? There is an issue https://github.com/notaryproject/.github/issues/70 already proposed archive the notary repo.

This will relax unnecessary workloads to org maintainers going forward. We add new maintainers to those repos indicate that new maintainers are responsible for its continuous maintenance for users. FYI the archival process below from the [archival process doc](https://github.com/notaryproject/.github/blob/main/REPOSITORIES.md#archiviation):

> The repository-level maintainers should be noticed about this archiviation issue. Once the proposal issue has a supermajority (two-thirds) of approval from Notary Project governance maintainers and there is no response from the repository-level maintainers after 30 days notice, the repository can be archived.”